apiserver/storage/storage.go

```diff
@@ -200,3 +201,69 @@ func (a *API) checkProviderTypeSupported(providerType storage.ProviderType) erro
 	}
 	return nil
 }
+
+var getVolumeManager = func(vs volume.VolumeState) volume.VolumeManager {
+	return volume.NewVolumeManager(vs)
+}
+
+func (a *API) ListVolumes(filter params.StorageVolumeFilter) (params.StorageVolumesResult, error) {
+	volumeManager := getVolumeManager(a.storage)
+
+	all, err := volumeManager.List()
+	if err != nil {
+		return params.StorageVolumesResult{}, err
+	}
+
+	if len(all) < 1 {
```

Comment:
I m checking if no volumes were found. Is there a better way to check if a collection (array/slice) is empty?